### PR TITLE
WIB-179 implement POST fixes

### PR DESCRIPTION
**Pull request by skaymen**
implement fixes requested in https://internal.cida.usgs.gov/jira/browse/WIBEACHES-179

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -209,6 +209,11 @@
             <artifactId>spring-security-web</artifactId>
             <version>5.0.0.RELEASE</version>
         </dependency>
+        <dependency>
+            <groupId>org.apache.commons</groupId>
+            <artifactId>commons-lang3</artifactId>
+            <version>3.1</version>
```
Comment (dsteinich) [inline pom.xml]: The current version is 3.7

File: src/main/java/gov/usgs/wim/wdnr/dao/StringBooleanTypeHandler.java [hunk 2]
```diff
@@ -11,7 +11,7 @@
 public class StringBooleanTypeHandler implements TypeHandler<Boolean> {
 
     public static final String TRUE = "Y";
-    public static final String FALSE = "N";
+    public static final String FALSE = null;
```
Comment (dsteinich) [inline src/main/java/gov/usgs/wim/wdnr/dao/StringBooleanTypeHandler.java]: Is it possible to combine this with the StringBooleanYNTypeHandler? missingRequiredFlag is using this handler and I am not sure that is appropriate (not in the JIRA list of Y/null columns).

My hunch is that we can preserve the null value and have the same handle process Y/N/null. (Unless missingRequiredFlag and colorChange need nulls converted to N.)

File: pom.xml (hunk 1 above)
Comment (skaymen) [reply pom.xml]: fixed

File: src/main/java/gov/usgs/wim/wdnr/dao/StringBooleanTypeHandler.java (hunk 2 above)
Comment (skaymen) [reply src/main/java/gov/usgs/wim/wdnr/dao/StringBooleanTypeHandler.java]: resolved

**Pull request merged**